### PR TITLE
Alertmanager configmap

### PR DESCRIPTION
**Pull request by vparekh94**
@kaxil @ianstanton I tried working backwards from the conversation with the customer to get this added to the docs. I don't really understand what needed to be added that ultimately resolved their issue. Can you please take a look and let me know what needs to be changed here?

### CODE REVIEW

File: v0.10/ee-modifying-alertmanager.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: "Modifying Alertmanager"
+description: "Sending Alerts to external sources"
+date: 2018-10-12T00:00:00.000Z
+slug: "ee-modifying-alertmanager"
+---
+
+## Alertmanager
+
+Astronomer users [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to push platform level alerts from Prometheus. Currently, these alerts can be subscribed to by email in the UI. The alertmanager `alert-manager-configmap.yaml` file can be modified to push alerts to external sources.
```
Comment (kaxil) [inline v0.10/ee-modifying-alertmanager.md]:
```suggestion
Astronomer users [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to push platform level alerts from Prometheus. Currently, these alerts can be subscribed to by email in the UI. The alertmanager `alertmanager-configmap.yaml` file can be modified to push alerts to external sources.
```

File: v0.10/ee-modifying-alertmanager.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+### Sending Alerts to Slack
+To add Slack as a destination for platform alerts, navigate to `helm.astronomer.io/charts/alertmanager/templates/alertmanager-configmap.yaml` and add the necessary configurations to for a Slack channel:
```
Comment (kaxil) [inline v0.10/ee-modifying-alertmanager.md]:
```suggestion
To add Slack as a destination for platform alerts, navigate to `helm.astronomer.io/charts/alertmanager/templates/alertmanager-configmap.yaml` and add the necessary configurations for a Slack channel:
```

File: v0.10/ee-modifying-alertmanager.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      {{ end }}
+- name: blackhole-receiver
+# Deliberately left empty to not deliver anywhere.
+- name: default-receiver
+  webhook_configs:
+  - url: http://astronomer-houston:8871/v1/alerts
+    send_resolved: true
+```
+
+Substitute the appropriate values for the channel and api_url and save the file.
```
Comment (kaxil) [inline v0.10/ee-modifying-alertmanager.md]:
```suggestion
Substitute the appropriate values for the `<channel>` and `<api_url>` and save the file.
```

**Comment by vparekh94** [comment]
@kaxil I definitely think we should advise customers a bit on how to manage their configurations with Terraform. However, most of our docs are assuming the customer is using helm so I wrote this one for that.

Would love to help with more terraform docs, but unfortunately a bit out of my expertise.

**Comment by kaxil** [comment]
> @kaxil I definitely think we should advise customers a bit on how to manage their configurations with Terraform. However, most of our docs are assuming the customer is using helm so I wrote this one for that.
> 
> Would love to help with more terraform docs, but unfortunately a bit out of my expertise.

I love doc improvements :) Thanks man

**Comment by kaxil** [comment]
I don't have merge permissions, @vparekh94 can you merge this one?

**Comment by petedejoy** [comment]
Is this ready to be merged @vparekh94 @kaxil?

**Comment by kaxil** [comment]
> Is this ready to be merged @vparekh94 @kaxil?

Yes

**Comment by kaxil** [comment]
I don't have permissions to merge this, can someone with permission merge this one @ianstanton @petedejoy

**Comment by petedejoy** [comment]
Got you! Sorry for the delay here- thanks for writing this up!

**Comment by petedejoy** [comment]
@vparekh94 should this go in the docs nav anywhere?

**Comment by vparekh94** [comment]
Guys I am super sorry, completely dropped the ball on responding to comments here

**Comment by samblackk** [comment]
@vparekh94 Mind pushing a hotfix branch to add this to the nav? I think it's breaking because it's missing.

**Pull request merged**